runtime/literal.go

```diff
@@ -82,7 +81,7 @@ func ParseLiteralArgumentList(
 	parameterCount := len(parameterTypes)
 
 	if argumentCount != parameterCount {
-		return nil, fmt.Errorf(
+		return nil, parser.NewUnpositionedSyntaxError(
```

Comment:
Are we planning on. going back to add positions here (and to the other unpositioned errors generally)? Errors that don't tell you their source code location are pretty unhelpful.

Reply:
For some reason, this `ParseLiteralArgumentList` method is only used within tests. So the position info is not available. Not sure why it is not in test-utils or something like that (maybe it was used in the parser before, but not anymore?), but I just let it be.